ng2-material/components/data_table/README.md

```diff
@@ -39,6 +39,7 @@ MdDataTable is an enhancment of classic data tables.
 | --- | --- | --- | --- |
 | md-data-table-selectable-row | tbody tr | boolean | Enable a checkbox for this row. |
 | selectable-value | tbody tr | string | value of the checkbox. If it's not set the checkbox's value will be the index of the row. |
+| is-active | tbody tr | boolean | initial value for  the checkbox. If true the row will be selected. |
```

Comment:
I know it's a bit late to complain, but the "is active" verbiage always bordered on confusing to me. We're really talking about whether a row/datum is _selected_, but I realize changing it now would constitute a breaking change. :disappointed: Same with select _able_ - its not the _able -- ability_ that is changing it's the select _ion_ state. I guess _selectable_ is a tie to the name of the component though.